config/areas/site/dialogs.php

```diff
@@ -471,8 +471,8 @@
 	],
 
 	'page.fields' => [
-		...$fields['model'],
 		'pattern' => '(pages/.*?)/fields/(:any)/(:all?)',
```

Comment:
`Question:` Shouldn't the last `(:all?)` be more specific to only match field routes that start with `dialogs/`? Not new with this PR but I think all of those field routes should maybe be more specific to really just match their use cases?